src/main/java/org/codehaus/mojo/license/AbstractAddThirdPartyMojo.java

```diff
@@ -559,6 +561,12 @@
      * @since 1.4
      */
     private Map<String, String> globalKnownLicenses;
+    
+    boolean isAggregatorBuild;
+    
+    Map<String, List<Dependency>> reactorProjectDependencies;
```

Comment:
Please explain in JavaDoc what is this field good for.

Reply:
Will do